### PR TITLE
Update to match latest designs

### PR DESCRIPTION
**Pull request by donaldkibet**
### What does this PR do?

Update the login screen to match the latest designs
https://app.zeplin.io/project/5f7223cfda10f94d67cec6d0/screen/601827beb2ba880451a7ee44

### Screenshoot
Feb-05-2021 14-05-22-min

﻿

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "@openmrs/esm-error-handling": "3.x",
     "@openmrs/esm-react-utils": "3.x",
     "carbon-components": "^10.21.0",
+    "@carbon/icons-react": "^10.25.0",
```
Comment (FlorianRappl) [inline package.json:53]: These are not shared afaik. We only share the framework-independent carbon packages.

File: src/location-picker/location-picker.component.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+.location-card {
+  max-width: 60rem;
+}
+
+.center {
+  text-align: center;
+}
+
+.location-submit-btn {
+  width: 21.75rem;
```
Comment (FlorianRappl) [inline src/location-picker/location-picker.component.scss:31]: Should we maybe introduce a `max-width: 100%`, too? The value seems to be rather arbitrary as well.

File: src/login/login.component.test.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { performLogin } from "./login.resource";
 import { setSessionLocation } from "../choose-location/choose-location.resource";
 import { useCurrentUser } from "../CurrentUserContext";
 import renderWithRouter from "../test-helpers/render-with-router";
+import userEvent from "@testing-library/user-event";
```
Comment (FlorianRappl) [inline src/login/login.component.test.tsx:9]: Forgot to install this package?

File: webpack.config.js [hunk 4]
```diff
@@ -31,15 +40,11 @@ module.exports = {
...
-        use: [
-          { loader: "style-loader" },
-          {
-            loader: "css-loader",
-            options: {
-              modules: true,
-            },
-          },
-        ],
+        use: ["style-loader", cssLoader],
```
Comment (FlorianRappl) [inline webpack.config.js:43]: I think the `modules` option is not on by default. Intentional to remove it?

**Review by FlorianRappl — COMMENTED** [review]
Great changes - thanks! :beers:

File: src/login/login.component.test.tsx (hunk 3 above)
Comment (donaldkibet) [reply src/login/login.component.test.tsx:9]: Installed, added to package.json

File: webpack.config.js (hunk 4 above)
Comment (donaldkibet) [reply webpack.config.js:41]: restored the module option

File: package.json [hunk 5]
```diff
@@ -64,12 +65,15 @@
     "@babel/preset-env": "^7.9.0",
     "@babel/preset-react": "^7.9.4",
     "@babel/preset-typescript": "^7.9.0",
+    "@carbon/icons-react": "^10.25.0",
```
Comment (FlorianRappl) [inline package.json:68]: Double mention - is already in `dependencies`.

**Review by FlorianRappl — COMMENTED** [review]
LGTM! :beers: (only one minor thing left)

**Review by FlorianRappl — APPROVED** [review]
Great update!

**Pull request merged**